src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -1357,7 +1377,7 @@ internal static Attribute GetAttribute(AttributeAst attributeAst)
             if (attributeType == null)
             {
                 throw InterpreterError.NewInterpreterException(attributeAst, typeof(RuntimeException), attributeAst.Extent,
-                    "CustomAttributeTypeNotFound", ParserStrings.CustomAttributeTypeNotFound, attributeAst.TypeName.FullName);
+                    "TypeNotFound", ParserStrings.TypeNotFound, attributeAst.TypeName.FullName);
```

Comment:
My comment was for a different place in the code and I stand by that comment. This location should not be changed.